x-pack/legacy/plugins/siem/server/lib/note/saved_object.ts

```diff
@@ -196,10 +200,7 @@ export class Note {
       request[internalFrameworkRequest]
     );
 
-    const savedObjects = await savedObjectsClient.find({
-      type: noteSavedObjectType,
-      ...options,
-    });
+    const savedObjects = await savedObjectsClient.find(options);
```

Comment:
@XavierM making `type` required in SavedObjectsFindOptions means we either have to include the type in every `getAllSavedNote()` call, or change the signature of `getAllSavedNote()`'s options parameter to not be `SavedObjectsFindOptions`.